app.json

```diff
@@ -2,9 +2,20 @@
   "name": "api.mozillascience.org",
   "description": "API server for science.mozilla.org",
   "addons": [
-    "heroku-postgresql"
+    "heroku-postgresql:hobby-dev"
   ],
   "scripts": {
-    "postdeploy": "python app/manage.py migrate"
+    "postdeploy": "sh scripts/postdeploy.sh"
+  },
+  "env": {
+    "DJANGO_ENV": "review",
```

Comment:
Hmm, it doesnot seem like app.json is used on staging and production (except when we create a new app from deploy on heroku button)